.github/workflows/backend.yml

```diff
@@ -0,0 +1,53 @@
+name: Backend
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "shop/**"
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - "shop/**"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Build frontend distribution for backend
+        run: yarn run build:dist
```

Comment:
This is a slow step. Is that necessary?
It takes a while to build (several minutes on my host).

Reply:
Yeah... Some of the tests reliy on having the shop build available.  We might be able to optimize later by adding this as artifacts.  I don't think the tests really care that the build is up to date.